src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -136,18 +137,27 @@ private static String generateCvrSnapshotId(String cvrId, Integer round) {
         : String.format("ballot-%s", cvrId);
   }
 
-  private static String getCdfIdForCandidateCode(String code) {
-    String id = candidateCodeToCdfId.get(code);
+  // generates an internal ContestSelectionId based on a candidate code
+  private static String getCvrContestSelectionIdForCandidateCode(String code) {
```

Comment:
I thought I commented on this before, but I don't see my comment. This and the next method (and the maps they use) should be labeled so that it's clear they're CDF-specific.